input/pagecontent/export.md

```diff
@@ -249,33 +249,57 @@ To obtain new and updated resources for patients in a group, as well as all data
+
+The _typeFilter parameter enables finer-grained filtering out of resources in the bulk data export response that would have otherwise been returned. For example, a client may want to retrieve only active prescriptions rather than all prescriptions and only laboratory observations rather than all observations. 
 
-The community has identified use cases for finer-grained, client-specified filtering. For example, some clients may want to retrieve only active prescriptions rather than historical prescriptions or only laboratory observations rather than all observations. 
+The value of the `_typeFilter` parameter is a comma-separated list of FHIR REST API queries. Resources with resource types specified in this query that would have been included in the bulk output SHALL NOT be returned (see [processing model](#processing-model)). FHIR [search result parameters](https://www.hl7.org/fhir/search.html#modifyingresults) (such as _sort, _include, and _elements) SHALL NOT be used as `_typeFilter` criteria. Clients should consult the server's capability statement to identify supported search parameters (see [server capability documentation](#server-capability-documentation)). 
 
-To request finer-grained filtering, a client MAY supply a `_typeFilter` parameter alongside the `_type` parameter. The value of the `_typeFilter` parameter is a comma-separated list of FHIR REST API queries that restrict the results of the export. FHIR search response parameters such as `_include` and `_sort` SHALL NOT be used. Since support for `_typeFilter` is OPTIONAL for a FHIR server, clients SHOULD be robust to servers that ignore `_typeFilter`. A client MAY repeat the `_typeFilter` parameter multiple times in a kick-off request. When repeated, the server SHALL treat the repeated values as if they were comma delimited values within a single `_typeFilter` parameter.
+A client MAY repeat the `_typeFilter` parameter multiple times in a kick-off request. When more than one `_typeFilter` parameter is provided with a REST query for the same resource type, the server SHALL include resources of that resource type that meet the criteria in any of the parameters (a logical "or").  
 
-##### Example Request
+Since support for `_typeFilter` is optional for a FHIR server, clients SHOULD be robust to servers that ignore `_typeFilter`.
 
-The following is an export request for `MedicationRequest` resources and `Condition` resources, where the client would further like to restrict `MedicationRequests` to requests that are `active`, or else `completed` after July 1, 2018. This can be accomplished with two subqueries joined together with a comma for a logical "or":
+**Example Request**
+
+The following is an export request for `MedicationRequest` resources, where the client would further like to restrict `MedicationRequests` to requests that are `active`, or else `completed` after July 1, 2018. This can be accomplished with two `_typeFilter` query parameters ():
```

Comment:
Did you intend to remove _type from the example? If so, wouldn't return other resources too?

Reply:
Yeah, don't think I meant to do that - will add it back.